chain/network/Cargo.toml

```diff
@@ -19,7 +19,7 @@ byteorder = "1.2"
 lazy_static = "1.4"
 tracing = "0.1.13"
 
-borsh = "0.6.1"
+borsh = "0.6.2"
```

Comment:
Cannot we have borsh version global per-project?

Reply:
What do you mean?

Reply:
I meant we keep the same version of the library in many places, and would be cool to set and update in one place.

Reply:
Let's do this in a separate PR for sure.